applications/blogs/backend/src/main/resources/libs/kyanite/blogs/components/quote/dialog/.content.json

```diff
@@ -0,0 +1,50 @@
+{
+  "sling:resourceType": "wcm/dialogs/dialog",
+  "tabs": {
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "generalTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "quoteText": {
+        "sling:resourceType": "wcm/dialogs/components/richtext",
+        "name": "quoteText",
+        "label": "Quote text",
+        "required": true,
+        "ws:disallowedContext": ["edit:panel"],
+        "configuration": "kyanite/common/components/common/rte/configurations/default"
+      },
+      "authorContainer": {
+        "sling:resourceType": "wcm/dialogs/components/container",
+        "authorName": {
+          "sling:resourceType": "wcm/dialogs/components/textfield",
+          "name": "authorName",
+          "label": "Author name",
+          "required": true
+        },
+        "authorCompany": {
+          "sling:resourceType": "wcm/dialogs/components/textfield",
+          "name": "authorCompany",
+          "label": "Author company"
+        },
+        "authorPhotoPath": {
+          "sling:resourceType": "wcm/dialogs/components/pathpicker",
+          "rootPath": "/content",
+          "name": "authorPhotoPath",
+          "description": "Author square photo asset (optional)",
+          "label": "Author photo",
+          "ws:disallowedContext": [
+            "edit:panel"
```

Comment:
because of [side panel extra save-properties request](https://teamds.atlassian.net/browse/WS-3150) . It affects path pickers and, probably, rte (not sure about that)

Reply:
I don't think so. We have a RTE used in other components as well.